.gitignore

```diff
@@ -7,7 +7,7 @@ TestData/EmailWithNonUnicodeSubject.eml
 *.pyc
 .pytest_cache
 
-CommonServerPython.py
+./CommonServerPython.py
```

Comment:
This is not good. As part of the way we run unit tests and linting we copy the common server file to the package dir. These files now appear in git status. I suggest we stay with the way it was. You can still commit the CommonServerPython file for the package with `--force`.

Reply:
The copying shouldn't add them to the git status. I think this way is better because we will keep working regularly with CommonServerPython as with all the other files in the system